scorecard/profile_data.py

```diff
@@ -7,22 +8,22 @@
 
 class MuniApiClient(object):
     def __init__(self, geo_code):
-        self.API_URL = 'https://data.municipalmoney.org.za/api/cubes/'
```

Comment:
I think this should be self.API_URL = settings.API_URL, rather than using settings inside the body of the class's methods.